server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexMediumTest.java

```diff
@@ -645,6 +652,76 @@ public void filter_by_created_at() throws Exception {
   }
 
   @Test
+  public void facet_on_created_at() throws Exception {
```

Comment:
Different test cases should be created, instead of testing all use cases in a single test